wifiphisher/pywifiphisher.py

```diff
@@ -925,6 +925,9 @@ def run():
 
     time.sleep(3)
 
+    #no longer need mac_matcher
+    mac_matcher.unbind()
```

Comment:
I did  `del self.mac_to_vendor` on purpose since I believe that is incorrect for someone to call `.getVendor` once `.unbind` is called and thus it should result to an error ("Errors should never pass silently."). I wanted to throw a custom/more relevant error but I didn't want to over-engineer it.

I am ok do it with the `unknown` way. Tell me what you think.